algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -373,6 +373,22 @@ public Integer getDistinct() {
         return null;
     }
 
+    private static final String KEY_FACETING_AFTER_DISTINCT = "facetingAfterDistinct";
+
+    public @Nullable Boolean getFacetingAfterDistinct() {
+        return parseBoolean(get(KEY_FACETING_AFTER_DISTINCT));
+    }
+
+    /**
+     * Force faceting to be applied after de-duplication. Please check <a href="https://www.algolia.com/doc/rest-api/search/#facetingafterdistinct">documentation</a> for consequences and limitations
+     *
+     * @param enabled if {@code true}, facets will be computed after de-duplication is applied.
+     * @see <a href="https://www.algolia.com/doc/rest-api/search/#facetingafterdistinct">facetingAfterDistinct's documentation</a>
```

Comment:
@clement-leprovost : WDYT of this? Does it add value to reference the parameters' documentation with `@see`?